VSRAD.Package/Commands/BreakModeDropdownCommand.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.VisualStudio.OLE.Interop;
+using Microsoft.VisualStudio.ProjectSystem;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using VSRAD.Package.ProjectSystem;
+using VSRAD.Package.ToolWindows;
+
+namespace VSRAD.Package.Commands
+{
+    [Export(typeof(ICommandHandler))]
+    [AppliesTo(Constants.RadOrVisualCProjectCapability)]
+    public sealed class BreakModeDropdownCommand : ICommandHandler
+    {
+        private readonly IProject _project;
+
+        [ImportingConstructor]
+        public BreakModeDropdownCommand(IProject project)
+        {
+            _project = project;
+        }
+
+        public Guid CommandSet => Constants.BreakModeDropdownCommandSet;
+
+        public OLECMDF GetCommandStatus(uint commandId, IntPtr commandText) =>
+            OLECMDF.OLECMDF_ENABLED | OLECMDF.OLECMDF_SUPPORTED;
+        public void Execute(uint commandId, uint commandExecOpt, IntPtr variantIn, IntPtr variantOut)
+        {
+            if (commandId == Constants.BreakModeDropdownListId)
+            {
+                if (variantOut != IntPtr.Zero) /* list available items */
+                {
+                    Marshal.GetNativeVariantForObject(Utils.BreakModeConverter.BreakModeOptions, variantOut);
+                }
+            }
+            if (commandId == Constants.BreakModeDropdownId)
+            {
+                if (variantOut != IntPtr.Zero) /* get current item */
+                {
+                    var currentMode = Utils.BreakModeConverter.BreakModeToString(_project.Options.DebuggerOptions.BreakMode);
```

Comment:
Suggestion: add a separate array for "short" break mode strings (`{ "Round-robin", "Rerun line",  "Multiple" }`) and return them for `Constants.BreakModeDropdownId`. This way, you can set `defaultWidth` for the dropdown in .vsct to 90 instead of 240 without losing information: 
![image](https://user-images.githubusercontent.com/32609527/125588425-0164a59c-afcd-4ff9-b967-0e0613660b53.png)